docker-tools/Dockerfile-from-travis

```diff
@@ -1,14 +1,15 @@
 #/* vim: set filetype=dockerfile : */
-FROM travisci/ci-garnet:packer-1512502276-986baf0
+FROM travisci/ci-garnet:packer-1515445631-7dfb2e1
 
 ADD . /tmp/bokeh
 RUN chown -R travis /tmp/bokeh
 
 USER travis
 
 ENV NO_GIT_FETCH=1
-ENV PATH="/home/travis/miniconda/bin:${PATH}"
-ENV CONDA_REQS="conda=4.4.11 conda-env=2.6.0 conda-build=3.0.27"
+ENV PATH="/home/travis/miniconda/bin/:${PATH}"
+ENV PATH="/home/travis/miniconda/envs/testenv/bin/:${PATH}"
```

Comment:
It avoids an extra image layer to combine these in to one statement, IIUC

Reply:
Indeed